dashboard/app/models/pd/workshop.rb

```diff
@@ -55,6 +55,7 @@ class Pd::Workshop < ApplicationRecord
 
   serialized_attrs [
     'fee',
+    'time_zone',
```

Comment:
Do we want this as a column or do we think serialized_attr is good? (I think it's good as-is, just thinking out loud)